docs/sources/flow/install/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Install
+weight: 50
+---
+
+# Install Grafana Agent Flow
+
+Distributions of Grafana Agent Flow are available for:
+
+{{< section >}}
```

Comment:
Should we also add a simpler section about someone just wanting to run the binary, without setting it up as a service and point them to our releases page?

Reply:
Yeah, I considered it at first and decided against it, but I think I'm back to it being a good idea. Will write this.